src/ast/stmt/PostConditionalStmt.php

```diff
@@ -1,73 +0,0 @@
-<?php
```

Comment:
what happens with `when`?

Reply:
Is replaced by `match`, which is a lot more powerful.